pkg/server/server.go

```diff
@@ -36,9 +36,9 @@ import (
 )
 
 const (
-	invalidTrustDomainAttestedNode = "An attested node with trust domain '%v' has been detected, " +
+	invalidTrustDomainAttestedNode = "an attested node with trust domain '%v' has been detected, " +
```

Comment:
This is also used to log, of which our convention is leading uppercase...

Reply:
Ah, go lint is complaining wherever logs begin with uppercase

Reply:
Yeah, and rightly so :)

Reply:
Also, the two below it begin with lower case...